tests/installation/zdup.pm

```diff
@@ -131,6 +133,12 @@ sub run() {
         elsif ($out =~ $zypper_dup_finish) {
             last;
         }
+        elsif ($out =~ $zypper_retrieving or $out =~ $zypper_check_conflicts) {
+            # probably to avoid hitting black screen on video
+            send_key 'shift', 1;
+            # continue but do not drop zypper_dup_fileconflict check
```

Comment:
Then it's me :) a comment to not remove zypper_dup_fileconflict check in a block that did not even care fo zypper_dup_fileconflict sounds strange... having checked some valued in an if, then claiming that others not tested are not touched sounds like it should be expectable....